jax_md/test_util.py

```diff
@@ -14,6 +14,8 @@
 
 """Defines testing utility functions."""
 
+import netCDF4 as nc
```

Comment:
I'm not sure how I feel about adding the netCDF dependency. On the one hand, it's pretty benign; on the other hand, now people will need netCDF to run the tests. What do you think of the idea of merging these test cases with the jammed packing test data I got from you and use in the pressure / stress tests (e.g. the JammingTestState data)? This would involve converting the states to a format that could be loaded with pure numpy.

Anyway, I don't feel so strongly, but this is just an idea.

Reply:
I'm not opposed to this. I'm personally on the "its pretty benign" side, but not strongly and am more than willing to change it to... I guess a text file? One question: would xarray be a less obnoxious dependency since its widely used?